tests/utilities/parsing.py

```diff
@@ -72,3 +90,8 @@ def test_lightning_setattr(tmpdir):
         lightning_setattr(m, 'learning_rate', 10)
         assert lightning_getattr(m, 'learning_rate') == 10, \
             'attribute not correctly set'
+
+    model5 = models[-1]
```

Comment:
```suggestion
    model5 = models[4]
```
same comment as above